skins/zwall.scss

```diff
@@ -0,0 +1,23 @@
+.zwap {
+  --primary-color: #FB0779;
+  --bg-color: #f7f7f7;
+  --text-color: #101832;
+  --link-color: #6F16FD;
+  --heading-color: #101832;
+  --border-color: #6F16FD;
+  --header-bg: #ffffff;
+  --block-bg: #ffffff;
+}
+
+@media (prefers-color-scheme: dark) {
+  .port {
```

Comment:
Should be `zwall` selector

Reply:
https://github.com/Zilliqa/snapshot-spaces/pull/30/commits/7d5afe4aef105a072fd4b65dea606305a043b26d